Dockerfile.cross

```
@@ -0,0 +1,16 @@
+# This image is meant to enable cross-architecture builds.
+# It assumes the ream binary has already been
+# compiled for `$TARGETPLATFORM` and moved to `./bin`.
+FROM --platform=$TARGETPLATFORM ubuntu:22.04
```

```suggestion
FROM ubuntu:22.04
```
--platform=$TARGETPLATFORM is a default behavior anyway.

---

I am fine with this as
- these is what reth does
- This is helpful for people unfamiliar with docker

---

> these is what reth does

While I don't agree with this and 

> This is helpful for people unfamiliar with docker

not sure about this one, I'm fine with this if this enhances readability for people unfamiliar with Docker. I'll leave this up to @ch4r10t33r

---

Ah I see you are just parroting the WARN if you try and build the Dockerfile, maybe say that next time xD